main/models.py

```diff
@@ -18,6 +19,7 @@ class AuditModel(TimestampedModel):
     """An abstract base class for audit models"""
 
     acting_user = ForeignKey(settings.AUTH_USER_MODEL, null=True, on_delete=CASCADE)
+    modified_by = CharField(default="", max_length=750, null=True, blank=True)
```

Comment:
Can it be renamed to traceback or call_back? It's a little confusing when I first saw it. Not a big deal, just a thought.